### ISSUE TITLE
this.declaration is null

### ISSUE THREAD

**Issue by aychedee** [css/csso#274]
Getting this stack trace in our CI process: 

```
 /src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/compress/Dimension.js:24
        if (this.declaration.property.name === 'flex') {
                            ^
TypeError: Cannot read property 'property' of null
    at Object.compressDimension (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/compress/Dimension.js:24:29)
    at Object.module.exports [as fn] (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/compress/index.js:17:29)
    at Object.walkAll (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/ast/walk.js:149:10)
    at List.each (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/utils/list.js:112:12)
    at Object.walkAll (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/ast/walk.js:122:27)
    at List.each (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/utils/list.js:112:12)
    at Object.walkAll (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/ast/walk.js:122:27)
    at Object.walkAll (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/ast/walk.js:65:25)
    at List.each (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/utils/list.js:112:12)
    at Object.walkAll (/src/fe/node_modules/gulp-csso/node_modules/csso/lib/compressor/ast/walk.js:58:24)
```

**Comment by lahmatiy** [comment]
It's already fixed (see #273). Please update to 1.6.1

**Comment by aychedee** [comment]
Cheers :)
